udata_ods/migrations/2023-07-12-remove-double-slash-resource.py

```diff
@@ -0,0 +1,38 @@
+
+'''
+Remove ODS resources that are duplicate of resource with one slash only
+'''
+import logging
+
+from udata.models import Dataset
+from udata.harvest.models import HarvestSource
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
```

Comment:
Is it normal that the `db` argument is never used?

Reply:
migrate functions expect this argument :)